lib/actions/project/tpl/package.json

```diff
@@ -10,6 +10,6 @@
   "license": "GPL-3.0",
   "dependencies": {
     "npx": "10.2.0",
-    "polkadot-deployer": "2.5.1"
+    "polkadot-deployer": "1.0.5"
```

Comment:
I think you should not merge update-v1 on master, but on v1, isn't it ?

Reply:
correct thx